callmetadata.schema.json

```diff
@@ -5,6 +5,27 @@
     "description": "Meta data supporting dialtone, ringdown, and shoutdown calls",
     "type": "object",
     "definitions": {
+        "state": {
+            "type": "object",
+            "properties": {
+                "state": {
+                    "type": "string",
+                    "description": "Name of the latest state of the current communication: CallOriginated, CallDelivered, CallEstablished, ect. Could be an enum with defined states. ",
```

Comment:
If it's not an enum we should remove this part of description: "Could be an enum with defined states."

I agree in not using enum at this time.